templates/terraform/examples/artifact_registry_repository_cmek.tf.erb

```diff
@@ -0,0 +1,9 @@
+resource "google_artifact_registry_repository" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+
+  location = "us-central1"
+  repository_id = "<%= ctx[:vars]['repository_id'] %>"
+  description = "<%= ctx[:vars]['description'] %>"
+  format = "DOCKER"
+  kms_key_name = "projects/my-project/locations/us-central1/keyRings/my-kr/cryptoKeys/my-key"
```

Comment:
This example becomes a test, so you'll want to make sure you also have a real kms key to reference in order for the test to pass. However, because they can't be deleted, our codebase uses a shared KMS key for running tests instead of creating those resources in the tests themselves. You'll want to do something similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/redis/terraform.yaml#L35-L37, except using `BoostrapKMSKey(t)`

Reply:
Thanks!